python/hawkey/nevra-py.c

```diff
@@ -222,19 +222,23 @@ to_query(_NevraObject *self, PyObject *args, PyObject *kwds)
 {
     PyObject *sack;
     DnfSack *csack;
-    if (!PyArg_ParseTuple(args, "O!", &sack_Type, &sack)) {
+    const char *kwlist[] = {"sack", "icase", NULL};
+    int icase = 0;
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O!|i", (char**) kwlist, &sack_Type, &sack,
```

Comment:
I prefer use `PyBool_Type` instead of integer for `icase`.
Somethink like this:
... `"O!|O!"`, (char**) kwlist, &sack_Type, &sack, &PyBool_Type, &py_icase);
gboolean icase = PyObject_IsTrue(py_icase);